packages/@react-aria/spinbutton/src/useSpinButton.ts

```diff
@@ -14,14 +14,17 @@ import {announce} from '@react-aria/live-announcer';
 import {AriaButtonProps} from '@react-types/button';
 import {HTMLAttributes, useCallback, useEffect, useRef} from 'react';
 import {InputBase, RangeInputBase, Validation, ValueBase} from '@react-types/shared';
+// @ts-ignore
+import intlMessages from '../intl/*.json';
+import {useMessageFormatter} from '@react-aria/i18n';
 
 
 export interface SpinButtonProps extends InputBase, Validation, ValueBase<number>, RangeInputBase<number> {
   textValue?: string,
-  onIncrement?: () => void,
-  onIncrementPage?: () => void,
-  onDecrement?: () => void,
-  onDecrementPage?: () => void,
+  onIncrement?: (isChained?: boolean) => void,
```

Comment:
old dead code that i forgot about

Reply:
ah, that's so stately knows if it should start from the currently parsed value vs the previous value in state
it's typically false, but when a increment/decrement button is held down and it starts spinning, then it needs to use the previous state value
it's typically false so that we start from whatever is parsed out of the numberfield (but isn't committed yet, so it's not in the state)